runner/templates/configmap.yaml

```diff
@@ -64,7 +64,8 @@ data:
       {{ if .Values.cache.enabled -}}
       [runners.cache]
         Type = {{ .Values.cache.type | quote }}
-        ServerAddress = "{{ template "minio.fullname" . }}:9000"
+        ServerAddress = "{{ template "minio.fullname" . }}-svc:9000"
```

Comment:
This line will break cases when `.Values.cache.serverAddress` is defined in config. What you want, is to change only the generated service name inside the template.

https://github.com/lwolf/kube-charts/blob/master/runner/templates/_helpers.tpl#L51